chart/values.schema.json

```diff
@@ -1785,7 +1785,7 @@
                             "default": "100Gi"
                         },
                         "storageClassName": {
-                            "description": "If using a custom StorageClass, pass name ref to all StatefulSets here.",
+                            "description": "If using a custom StorageClass, pass name ref to all StatefulSets here (can be templated).",
```

Comment:
This is used for the storage class of the scheduler as well as you can see in [`scheduler-deployment.yaml`](https://github.com/Aakcht/airflow/blob/patch-4/chart/templates/scheduler/scheduler-deployment.yaml#L336) .

@romsharon98 think we should add something about this here?

Reply:
Please let me know if you think this should be added here, but I'm not sure it is needed since it can only affect scheduler when using local executor, so when scheduler pod is also a worker.